.github/workflows/deploy_aws.yml

```diff
@@ -12,13 +12,30 @@ env:
   ECS_CLUSTER: vacationplanner
   ECS_TASK_DEFINITION: vacation_stay_planner_definition
   CONTAINER_NAME: vacation_stay_planner
+  DIRECTORY: vacation_stay_scrapper,flight-scrapper-service
 
 
 #permissions:
 #  contents: read
 
+
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ${{fromJson(env.DIRECTORY)}}
```

Comment:
Please explain what does this strategy

Reply:
Using an array listing those directories can automate and simplify this process.